### DISCUSSION TITLE
About Dimensionality

### DISCUSSION THREAD

**Discussion by AGVrahatis** [panagiotisanagnostou/HiPart#8] [Q&A] [answered]
Congrats on your package. Should I run it on data with more than 300K dimensions? I have some methylation data with an ultra-high volume of features.

**Comment by panagiotisanagnostou** [comment, marked as answer]
Thank you very much, and Ι even more, thank you for your question.

Most of the algorithms in our package use a projection of the data in its first principal component to determine the dissimilarity of the data (because we are talking about discriminative clustering). They then process the data on the principal component to split it. This process circumvents the curse of dimensionality that data of too many dimensions like the data you describe to me suffer.

So to answer your question, yes, you can run the PDDP, dePDDP, iPDDP, and kM-PDDP packet algorithms on your data.

**Reply by AGVrahatis** [reply]
Thank you for your feedback.
